packages/ffe-form/less/field-message.less

```diff
@@ -6,6 +6,7 @@
     grid-template-columns: auto 1fr;
     grid-column-gap: @ffe-spacing-2xs;
     font-size: var(--ffe-fontsize-body-text);
+    line-height: 1.5rem;
```

Comment:
Jeg mener at vi uansett bør legge inn `line-height` på `.ffe-field-message`. Nå bestemmes denne av konteksten, og vil være forskjellig avhengig av om man f.eks. har brukt `.ffe-body-text` eller ikke. `1.5rem` er standard for i omtrent alle ffe-tekster.